NorthstarDedicatedTest/scriptsrson.cpp

```diff
@@ -37,15 +38,23 @@ void ModManager::BuildScriptsRson()
 				_coolscript.gnut
 			]*/
 
+			// check if the same script has been added already.
+			if (std::find(scriptsAdded.begin(), scriptsAdded.end(), script.Path) != scriptsAdded.end())
+			{
+				continue;
```

Comment:
This would silently drop the script. Consider notifying the user about this.
This could create issues if a mod is relying on modifying a file for the rest of its behaviour, so we need it be logged

Reply:
Removed this in favor of a dependency constant system.